<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1016</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1016</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1016</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="xLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kxLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>kx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kyLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ky</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alphaLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>alpha Â°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translatePushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scalePushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotatePushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoPushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoscalingCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoscaling</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
